app/views/docs/main.phtml

```diff
@@ -68,7 +68,7 @@ $getSpecsUrl = function($type, $platform) use ($latestVersion) {
 
     <p>Appwrite is a secure, open-source backend that abstracts the complexity of common, complex, and repetitive tasks required for building a modern app.</p>
 
-    <p>Appwrite provides you with a set of secure APIs, tools, and a management console UI to help you build your apps quickly without compromising security. Appwrite includes features like user authentication and account management, user preferences, database and file storage, cloud functions, localization, image manipulation, and more.</p>
+    <p>Appwrite provides you with a set of secure APIs, tools, and a management console UI to help you build your apps quickly without compromising security. You can add user authentication and management, data and file storage, run server-side code, image manipulation, localization, and more into your apps with Appwrite.</p>
```

Comment:
I rewrote this from the perspective of a developer who has no idea what a BaaS is.

Instead of features, let's just tell them what can be added into their apps with Appwrite.